lib/utility/api/module-list.ts

```diff
@@ -1,4 +1,5 @@
 // Utility side modules, please sort alphabetically.
 export const utilityNodeModuleList: ElectronInternal.ModuleEntry[] = [
-  { name: 'net', loader: () => require('./net') }
+  { name: 'net', loader: () => require('./net') },
+  { name: 'systemPreferences', loader: () => require('./system-preferences') }
```

Comment:
Can we just map this to the `@browser` file? no need to copy paste the impl I think

Reply:
done- uses @electron/internal/browser/api/system-preferences now. However, the net module and some others all re-copy: any insights as to why we chose to do that